content/master/cli/command-reference.md

```diff
@@ -32,6 +32,135 @@ Client Version: v1.16.0
 Server Version: v1.16.0
 ```
 
+## render 
+
+The `crossplane render` command previews the output of a 
+[composite resource]({{<ref "../concepts/composite-resources">}}) after applying 
+any [composition functions]({{<ref "../concepts/compositions">}}).
+
+{{< hint "important" >}}
+The `crossplane render` requires you to use composition functions.
```

Comment:
```suggestion
The `crossplane render` command requires you to use composition functions.
```

Reply:
These command reference docs are all just relocated existing content, so these were all pre-existing issues. They're easy to fix up though so I've pushed them in a separate commit.